### ISSUE TITLE
cannot run new project. deno 1.9.2 aleph 0.2.28

### ISSUE THREAD

**Issue by m1cl** [alephjs/aleph.js#340]
Check https://deno.land/x/aleph@v0.2.28/cli/dev.ts
error: Uncaught (in promise) TypeError: TS2345 [ERROR]: Argument of type 'TransformerFactory<SourceFile>' is not assignable to parameter of type 'TransformerFactory<SourceFile> | CustomTransformerFactory'.
  Type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").TransformerFactory<import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").SourceFile>' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").TransformerFactory<import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").SourceFile>'.
    Types of parameters 'context' and 'context' are incompatible.
      Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").TransformationContext' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").TransformationContext'.
        The types of 'factory.createTypeParameterDeclaration(...).parent' are incompatible between these types.
          Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").DeclarationWithTypeParameterChildren | import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").InferTypeNode' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").DeclarationWithTypeParameterChildren | import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").InferTypeNode'.
            Type 'JSDocTemplateTag' is not assignable to type 'DeclarationWithTypeParameterChildren | InferTypeNode'.
              Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").JSDocTemplateTag' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").JSDocTemplateTag'.
                Types of property 'parent' are incompatible.
                  Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").JSDoc | import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").JSDocTypeLiteral' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").JSDoc | import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").JSDocTypeLiteral'.
                    Type 'JSDoc' is not assignable to type 'JSDoc | JSDocTypeLiteral'.
                      Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").JSDoc' is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").JSDoc'.
                        Types of property 'parent' are incompatible.
                          Type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").HasJSDoc'
 is not assignable to type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").HasJSDoc'.
                            Type 'ImportEqualsDeclaration' is not assignable to type 'HasJSDoc'.
                              Property 'isTypeOnly' is missing in type 'import("https://cdn.esm.sh/v41/typescript@4.1.2/lib/typescript.d.ts").ImportEqualsDeclaration' but required in type 'import("https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts").ImportEqualsDeclaration'.
    if (reactRefresh) transformers.before!.push(reactRefreshTS())
                                                ~~~~~~~~~~~~~~~~
    at https://deno.land/x/aleph@v0.2.28/tsc/compile.ts:28:49

    'isTypeOnly' is declared here.
            readonly isTypeOnly: boolean;
                     ~~~~~~~~~~
        at https://cdn.esm.sh/v41/typescript@4.2.4/lib/typescript.d.ts:1601:18
    const { default: cmd } = await import(`./cli/${command}.ts`)
                             ^
    at async main (https://deno.land/x/aleph@v0.2.28/cli.ts:163:30)

**Comment by ije** [comment]
sorry, pls try the **alpha.33**, **0.2.28** only can work in deno 1.6.3. and the 0.3 beta will land in next few days, and i'm updating the docs.

**Comment by About7Sharks** [comment]
Can confirm that I ran into the same issue today

**Comment by Bili-Jun** [comment]
alpha.xx has a typeError:

deno install --unstable -A -f -n aleph https://deno.land/x/aleph@v0.3.0-alpha.33/cli.ts
aleph init hello
aleph dev

===>

error: Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.assign (<anonymous>)
    at https://deno.land/x/aleph@v0.3.0-alpha.32/server/helper.ts:24:8
    at <anonymous> (<anonymous>)

**Comment by Bili-Jun** [comment]
> alpha.xx has a typeError:
> 
> deno install --unstable -A -f -n aleph https://deno.land/x/aleph@v0.3.0-alpha.33/cli.ts
> aleph init hello
> aleph dev
> 
> ===>
> 
> error: Uncaught (in promise) TypeError: Cannot convert undefined or null to object
> at Function.assign ()
> at https://deno.land/x/aleph@v0.3.0-alpha.33/server/helper.ts:24:8
> at ()

➜ deno -V
deno 1.11.2
➜  hello aleph dev   
Check https://deno.land/x/aleph@v0.3.0-alpha.32/cli.ts
Check https://deno.land/x/aleph@v0.3.0-alpha.32/cli/dev.ts
ERROR Aleph.js needs Deno 1.8.0+, please upgrade Deno.

??????

**Comment by shadowtime2000** [comment]
@Bili-Jun That was already solved from Aleph alpha .32

**Comment by Sfxrlz** [comment]
It didnt work with either alpha 31/32 or 32 for me I ended up using alpha-30 now its working

**Comment by ije** [comment]
try beta.1 with deno 1.13

**Comment by Acen** [comment]
> @Acen this because your command missed `--unstable` flag since the old command didn't add it, try run:
> 
> ```shell
> deno run -A https://deno.land/x/aleph/install.ts
> ```

And we're working.

Cheers.

**Comment by talentlessguy** [comment]
can this be closed? latest 0.3 beta works fine on deno 1.14
